### PR TITLE
Simplify e2e tests

### LINKED ISSUE #112: Bug: Navigating to examples/extension-epic-mode throws an error and white page of death
## Description

URL: https://docs.remirror.org/examples/extension-epic-mode

I am seeing this in Chrome (75.0.3770.142) and Firefox (68 and 68.0.1 as well).

Screen Shot 2019-07-20 at 10 37 49 PM

Errors:

```
TypeError: "this.container.appendChild is not a function"
    value extension-epic-mode.esm.js:266
    view extension-epic-mode.esm.js:400
    updatePluginViews index.js:4476
    St index.js:4339
    v react-ssr.esm.js:116
    React 13
    safeSetState loadable.esm.js:164
    promise loadable.esm.js:223
react-dom.production.min.js:198:186
TypeError: "this.container.appendChild is not a function"
    value extension-epic-mode.esm.js:266
    view extension-epic-mode.esm.js:400
    updatePluginViews index.js:4476
    St index.js:4339
    v react-ssr.esm.js:116
    React 13
    safeSetState loadable.esm.js:164
    promise loadable.esm.js:223
react-dom.production.min.js:198:186
```

## Possible Fix

In general, a [react error boundary](https://reactjs.org/docs/error-boundaries.html) at the page-level could allow a single page to fail without causing a white page of death.

Specifically, I'm not exactly sure how to fix a missing container like this.

### PR DESCRIPTION
**Pull request by ifiokjr**
## Description

Improve e2e testing and allow failure to affect the build again.

Tests are now run by the server. So the test suite can choose.

- Fixes #112 - caused by an overzealous deep merge overwriting the canvas dom element container

## Checklist

- [x] I have read the [**CONTRIBUTING**](https://github.com/ifiokjr/remirror/blob/master/docs/contributing.md) document.
- [x] My code follows the code style of this project and `yarn fix` runs successfully.
- [x] New code is unit tested and all current tests pass when running `yarn test` .

### CODE REVIEW

File: @remirror/showcase/src/epic-mode.tsx [hunk 1]
```diff
@@ -31,6 +30,7 @@ const EpicModeComponent: FC<EpicModeComponentProps> = ({
   placeholder = 'Type for epic...',
   shake,
 }) => {
+  console.log(document.body);
```
Comment (charlex) [inline @remirror/showcase/src/epic-mode.tsx]: Just noting the `console.log`

File: support/jest/puppeteer.ts [hunk 2]
```diff
@@ -1,40 +1,19 @@
...
-    usedPortAction: 'kill',
-    launchTimeout: 120000,
-  },
-};
 
 let serverSetupPromise: Promise<void> | undefined;
 
 export async function setupServer(globalConfig: Config.GlobalConfig) {
-  await setup(__SERVERS__.map(server => serverConfig[server]));
+  console.log(server);
```
Comment (charlex) [inline support/jest/puppeteer.ts]: noting console.log

File: package.json [hunk 3]
```diff
@@ -33,7 +33,6 @@
     "docker:puppeteer:windows": "docker-compose -f ./support/docker/docker-compose.yml build puppeteer-windows",
     "docker:test:puppeteer": "docker-compose -f ./support/docker/docker-compose.yml run puppeteer-linux yarn test:puppeteer",
     "docker:test:puppeteer:firefox": "docker-compose -f ./support/docker/docker-compose.yml run puppeteer-linux yarn test:puppeteer:firefox",
-    "docs": "cd docs && yarn dev --debug",
```
Comment (charlex) [inline package.json]: [contributing.md](https://github.com/ifiokjr/remirror/blob/master/docs/contributing.md) will need to be updated. IS the new intended command `yarn build:docs` ?

File: @remirror/showcase/src/epic-mode.tsx (hunk 1 above)
Comment (ifiokjr) [reply @remirror/showcase/src/epic-mode.tsx]: Good spot 👍

File: package.json (hunk 3 above)
Comment (ifiokjr) [reply package.json]: Good point - it probably shouldn't be removed.

**Comment by ifiokjr** [comment]
@charlex thanks for the comments here btw. Helped me catch some issues.

**Pull request merged**